api/column_family.cc

```diff
@@ -973,18 +973,67 @@ void set_column_family(http_context& ctx, routes& r) {
         });
     });
 
+    cf::toppartitions_generic.set(r, [&ctx] (std::unique_ptr<request> req) {
+        std::vector<std::tuple<sstring, sstring>> names;
+
+        auto name_list = req->get_query_param("name_list");
+        stringstream ss { name_list };
+        std::string name;
+        while (!name_list.empty() && ss.good()) {
+            std::getline(ss, name, ',');
+            names.emplace_back(parse_fully_qualified_cf_name(name));
+        }
+
+        api::req_param<std::chrono::milliseconds, unsigned> duration{*req, "duration", 1000ms};
+        api::req_param<unsigned> capacity(*req, "capacity", 256);
+        api::req_param<unsigned> list_size(*req, "list_size", 10);
+
+        apilog.info("toppartitions query: #names={} duration={} list_size={} capacity={}",
+            names.size(), duration.param, list_size.param, capacity.param);
+
+        return seastar::do_with(db::toppartitions_query(ctx.db, std::move(names), duration.value, list_size, capacity), [&ctx](auto& q) {
```

Comment:
Can't we unified the implamentation for the specific and non specific CF?
if not anyting else, you can always call the general with the specific names, right?

Reply:
not exactly sure what you mean, can you elaborate a bit more?
Are you referring to the distinction between `toppartitions` and `toppartitions_generic`? Or something else?

Reply:
yes, it seems that there will be two implementation, one for a specific table and one for multiple tables